src/common/ui-wrapper.ts

```diff
@@ -27,7 +27,7 @@ class UIWrapper {
     } else {
       this._uiManager = new UIManager(player, config);
       if (config.customPreset || (window.kalturaCustomPresetMap && window.kalturaCustomPresetMap[config.targetId])) {
-        this._uiManager.buildCustomUI(config.customPreset || window.kalturaCustomPresetMap[config.targetId]['audioPreset']);
+        this._uiManager.buildCustomUI(config.customPreset || window.kalturaCustomPresetMap[config.targetId]['audioPlayer']);
```

Comment:
don't use a hardcoded string for the plugin name, take it from an object like you do in kaltura-player.ts

Reply:
but i need specific preset, here is the audio preset, how can i use it from object?